File: java/8 kyu/CountBy.java
// https://www.codewars.com/kata/5513795bd3fafb56c200049e/

import java.util.stream.IntStream;

public class CountBy {
  public static int[] countBy(int x, int n) {
    return IntStream.rangeClosed(1, n).map(numb -> numb * x).toArray();
  }
}

File: c/8kyu/count_by.c
//https://www.codewars.com/kata/5513795bd3fafb56c200049e
void count_by(unsigned x, unsigned n, unsigned result[n]) {
  for(int i = 0; i < n; i ++) {
    result[i] = x * (i + 1);
  }
}
